p2p/antiflood/factory/disabledAntiFlood.go

```diff
@@ -0,0 +1,31 @@
+package factory
+
+import (
+	"github.com/ElrondNetwork/elrond-go/p2p"
+)
+
+type disabledAntiFlood struct {
+}
+
+// ResetForTopic won't do anything
+func (n *disabledAntiFlood) ResetForTopic(topic string) {
```

Comment:
You can rename pointer receiver to something more appropriate

Reply:
done